src/client/Inode.cc

```diff
@@ -252,6 +255,7 @@ bool Inode::caps_issued_mask(unsigned mask, bool allow_impl)
       cap_is_valid(*auth_cap) &&
       (auth_cap->issued & mask) == mask) {
     auth_cap->touch();
+    session->cap_hit();
```

Comment:
For this case, shouldn't the session be the auth_cap's corresponding session, and also the same in LINE#267 for each Cap.

Reply:
`get_session()` returns auth_cap->session if auth_cap is valid.

Reply:
That means for one specified inode, it's auth_cap->session will be responsible for the hit/mis counting if it has a auth_cap, right ?

Reply:
Right. Are things different in the kclient implementation?

Reply:
Currently the approach in kclient is not graceful as expected, and will switch to the same approach as this.